Public/Src/Tools/Xldb.Proto/EnumsAndHelperStructs.proto

```diff
@@ -338,176 +350,58 @@ enum PipType{
     PipType_SealDirectory = 9;
 }
 
-enum SealDirectoryType{
-    SealDirectoryType_SealDirectory = 0;
-    CompositeSharedOpaqueDirectory = 1;
-}
-

```

Comment:
we don't use this enum?

Reply:
Nope, we only use it serialize/deserialize and dont store it in any member variable for SealDirectories.

---
In reply to: [320543153](https://github.com/microsoft/BuildXL/pull/820#discussion_r320543153) [](ancestors = 320543153)